### DISCUSSION TITLE
UMAP clusters for Multiomic data

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#776] [Questions / Documentation]
Although UMAP_ATAC, UMAP_RNA share the same Cluster numbering and and Colors are the cells that constitute each cluster different?
For example, are the cells in C1 of UMAP_ATAC different from those that make up C1 of UMAP_RNA in the Multiomic data?

Thanks!

**Comment by rcorces** [comment]
This depends on how you are coloring the cells and what value in `cellColData` is being used.
